code/datums/datacore.dm

```diff
@@ -185,6 +185,7 @@ GLOBAL_VAR_INIT(record_id_num, 1001)
 			G.fields["notes"] = H.gen_record
 		else
 			G.fields["notes"] = "No notes found."
+		G.fields["nt_relation"] = H?.client?.prefs?.active_character?.nanotrasen_relation || "Unknown relation."
```

Comment:
Maybe make the "Unknown Relation." text a define so we don't worry about them falling out of sync

Reply:
I think its fine without being a define